build-script.py

```diff
@@ -205,35 +209,17 @@ def generate_gyb_files(gyb_exec, verbose, add_source_locations, destination=None
                     verbose=verbose,
                 )
 
-    # Generate the new .swift files in a temporary directory and only copy them
-    # to Sources/SwiftSyntax/gyb_generated if they are different than the files
-    # already residing there. This way we don't touch the generated .swift
-    # files if they haven't changed and don't trigger a rebuild.
-    for gyb_file in os.listdir(swiftsyntax_sources_dir):
-        if not gyb_file.endswith(".gyb"):
-            continue
+    generate_gyb_files_helper(gyb_exec, SWIFTSYNTAX_DIR, swiftsyntax_destination,
+        temp_files_dir, add_source_locations, verbose)
 
-        gyb_file_path = os.path.join(swiftsyntax_sources_dir, gyb_file)
-
-        # Slice off the '.gyb' to get the name for the output file
-        output_file_name = gyb_file[:-4]
-
-        generate_single_gyb_file(
-            gyb_exec,
-            gyb_file_path,
-            output_file_name,
-            destination,
-            temp_files_dir,
-            add_source_locations,
-            additional_gyb_flags=[],
-            verbose=verbose,
-        )
+    generate_gyb_files_helper(gyb_exec, SWIFTSYNTAXBUILDER_DIR, 
+    swiftsyntaxbuilder_destination, temp_files_dir, add_source_locations, verbose)
```

Comment:
I think this line should be indented.
```suggestion
        swiftsyntaxbuilder_destination, temp_files_dir, add_source_locations, 
        verbose)
```